packages/server/src/Hocuspocus.ts

```diff
@@ -903,6 +909,24 @@ export class Hocuspocus {
   getMessageLogs() {
     return this.debugger.get()?.logs
   }
+
+  getDirectConnection({ documentName, context }: { documentName: string, context?: any }): DirectConnection {
```

Comment:
maybe the signature can just be

`getDirectConnection(documentName: string, context?: any): DirectConnection`

(or probably `openDirectConnection`? 🤔)

Reply:
Changed per your recommendation